app/Controllers/Http/DeletesController.ts

```diff
@@ -0,0 +1,27 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Profile from 'App/Models/Profile'
+
+export default class DeletesController {
+    public async deleteUser({ request, auth, response }: HttpContextContract) {
+        try {
+            const number = request.body().mobile_number
+            const user  = auth.use('api').user!
+    
+            const profile = await Profile.findByOrFail('user_id', user.id)
+            if(profile.mobileNumber === number) {
```

Comment:
Why can we add mobileNumber in where condition of query?

Reply:
Done